src/ol/sphere/sphere.js

```diff
@@ -58,6 +58,34 @@ ol.Sphere.prototype.cosineDistance = function(c1, c2) {
 
 
 /**
+ * Returns the area using Vincenty for a list of coordinates.
+ *
+ * [Reference](http://trs-new.jpl.nasa.gov/dspace/handle/2014/40409)
+ * Robert. G. Chamberlain and William H. Duquette, "Some Algorithms for
+ * Polygons on a Sphere", JPL Publication 07-03, Jet Propulsion
+ * Laboratory, Pasadena, CA, June 2007
+ *
+ * @param {Array.<ol.Coordinate>} coordinates List of coordinates.
+ * @return {number} Area.
+ * @api
+ */
+ol.Sphere.prototype.vincentyArea = function(coordinates) {
+  var area = 0;
+  var x1 = coordinates[coordinates.length - 1][0];
+  var y1 = coordinates[coordinates.length - 1][1];
+  for (var i = 0, ii = coordinates.length; i < ii; i++) {
```

Comment:
Minor, but if you assign `coordinates.length` to a variable before assigning `x1`, you can reuse it instead of dereferencing `coordinates.length` 3 times.